dev/com.ibm.ws.jsf.2.2_fat/.classpath

```diff
@@ -0,0 +1,52 @@
+<classpathentry kind="src" output="bin/main" path="test-applications/PI30335.jar/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/TestJSF22ViewAction.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/CDITests.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/BeanValidationTests.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/JSF22FlashEvents.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/JSF22FaceletsResourceResolverAnnotation.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/TestJSF22Ajax.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI47600.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI50108.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/CDIFacesFlows.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/JSF22FacesFlows.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI46218Flow1.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI46218Flow2.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI57255CDI.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI59422.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI63135.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/ViewScopedCDIBean.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/ViewScopedJSFBean.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI64718.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI64714.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI67525.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI85492.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PI90507.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/JSF22BackwardCompatibilityTests.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/JSF22HTML5.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/JSF22SimpleHTML.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/CDICommon/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/JSF22InputFile.war/src">
+<classpathentry kind="src" output="bin/main" path="test-applications/PH01566.war/src">
+<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8/"/>
+<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
```

Comment:
I'm not familiar with the gradleclasspathcontainer, is this used in other test projects? Usually I see `<classpathentry kind="con" path="aQute.bnd.classpath.container"/>`